source/extensions/key_value/file_based/config.cc

```diff
@@ -48,9 +48,9 @@ KeyValueStorePtr FileBasedKeyValueStoreFactory::createStore(
   const auto file_config = MessageUtil::anyConvertAndValidate<
       envoy::extensions::key_value::file_based::v3::FileBasedKeyValueStoreConfig>(
       typed_config.config().typed_config(), validation_visitor);
-  auto seconds =
-      std::chrono::seconds(DurationUtil::durationToSeconds(file_config.flush_interval()));
-  return std::make_unique<FileBasedKeyValueStore>(dispatcher, seconds, file_system,
+  auto milliseconds =
+      std::chrono::milliseconds(DurationUtil::durationToMilliseconds(file_config.flush_interval()));
```

Comment:
How annoying that this DurationUtil::durationToMilliseconds method doesn't return an std::chrono::milliseconds!